lib/server.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra/base'
+
+class ChitterChallenge < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.